src/test/groovy/com/ullink/NUnitPluginTest.groovy

```diff
@@ -1,6 +1,7 @@
 package com.ullink
```

Comment:
What about adding an integration-like test which executes the task on a packaged nunit result.xml resource and asserts the content of the generated HTML report?

Reply:
Will try to add one